apps/react-18-tests-v9/cypress.config.ts

```diff
@@ -1,3 +1,27 @@
+import * as path from 'path';
 import { baseConfig } from '@fluentui/scripts-cypress';
 
-export default baseConfig;
+const specs = [
+  path.resolve('../../apps/react-18-tests-v9/src/**/*.cy.{tsx,ts}'),
+  path.resolve('../../packages/react-components/**/library/src/**/*.cy.{tsx,ts}'),
+
+  // Exclude files that are not compatible with React 18 yet
+  '!' + path.resolve('../../packages/react-components/react-overflow/library/src/**/*.cy.{tsx,ts}'),
+  '!' + path.resolve('../../packages/react-components/react-tag-picker/library/src/**/*.cy.{tsx,ts}'),
+  '!' + path.resolve('../../packages/react-components/react-toolbar/library/src/**/*.cy.{tsx,ts}'),
```

Comment:
These are manually disabled as they are failing currently on React 18. A follow-up PR will be created with the fixes.

Reply:
nit: having a separate array which explicitly enums problematic projects and then generate the globs would be cleaner